docs/content/concepts/sui-move-concepts/packages/automated-address-management.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Automated Address Management
+---
+
+When you publish or upgrade a package, its address (also known as the package ID) is tracked in the `Move.lock` file. This bookkeeping is done automatically so that you can avoid recording or updating hex addresses (e.g., in the `Move.toml` file). 
+
+When you publish or upgrade your package on multiple chains (e.g., `mainnet`, `testnet`, `devnet`) separate addresses for each chain will be tracked for you. This tracking is based on your active environment (e.g., run `sui client active-env`). For example, if you set your active environment to an RPC connected to `testnet` and publish a package, the `Move.lock` will record the address for that package and associate it with your active environment (`testnet`).
+
+Note that automated address management works for one package published to one or more chains. When you publish or upgrade, it uses the contents of the package's working directory. If a package is republished to a chain, addresses tracked for the previously published package will be overwritten for that chain.
+
+## Adopting automated address management for published packages
+
+Previously a `published-at` entry was **mandatory** in the `Move.toml` file, which sets the address of the latest published package. It is no longer required if this data is tracked in the `Move.lock` file. For an existing package, you'll need to add the necessary data to the `Move.lock` file so that it can be automatically tracked:
+
+1. Switch to your active environment for the chain that your package is published on
+
+```
+sui client --switch --env <YOUR-CHAIN-ENVIRONMENT>
+```
+
+2. Run the `manage-package` command with the data for your published package
+
+```
+sui move manage-package --environment "$(sui client active-env)" \
+                        --network-id "$(sui client chain-identifier)" \
+                        --original-id 'ORIGINAL-ADDRESS' \
+                        --latest-id 'LATEST-ADDRESS' \
+                        --version-number 'VERSION-NUMBER'
+```
+
+- `ORIGINAL-ADDRESS` should be the address your package was first published at. If you never upgraded your package, this the same address as your `published-at` address in the `Move.toml`
+- `LATEST-ADDRESS` should be the latest package address. If you never upgraded your package, it is the same as `ORIGINAL-ADDRESS`. If you upgraded your package, this is the same address as your current `published-at` address in the `Move.toml`.
+- `VERSION-NUMBER` is `1` if you never upgraded your package. Otherwise, it is a number greater than `1` depending on how many times you upgraded your package. In this case, look up the package at `LATEST-ADDRESS` to determine the version number.
```

Comment:
```suggestion
      - `VERSION-NUMBER` is `1` if you never upgraded your package. Otherwise, it is a number greater than `1` depending on how many times you upgraded your package. In this case, look up the package at `LATEST-ADDRESS` to determine the version number.
```